ReactNativeClient/lib/joplin-renderer/MdToHtml.js

```diff
@@ -200,15 +200,14 @@ class MdToHtml {
 		//
 		// Using the `context` object, a plugin can define what additional assets they need (css, fonts, etc.) using context.pluginAssets.
 		// The calling application will need to handle loading these assets.
-
+		markdownIt.use(rules.copy_to_clipboard(context));
 		markdownIt.use(rules.image(context, ruleOptions));
 		markdownIt.use(rules.checkbox(context, ruleOptions));
 		markdownIt.use(rules.link_open(context, ruleOptions));
 		markdownIt.use(rules.html_image(context, ruleOptions));
 		if (this.pluginEnabled('katex')) markdownIt.use(rules.katex(context, ruleOptions));
 		if (this.pluginEnabled('fountain')) markdownIt.use(rules.fountain(context, ruleOptions));
 		if (this.pluginEnabled('mermaid')) markdownIt.use(rules.mermaid(context, ruleOptions));
-		markdownIt.use(rules.sanitize_html(context, ruleOptions));
 		markdownIt.use(rules.highlight_keywords(context, ruleOptions));
```

Comment:
Why is the `sanitize_html` deleted? Please add it back. That's probably the reason why the tests fail.

Reply:
I don’t know how that happened but I’ll add it back anyway